src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/UriExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
+{
+    internal static class UriExtensions
```

Comment:
I copied it this Microsoft.CodeAnalysis.Razror to avoid having to reference it. A bit ugly I guess. I'm not sure we are at the point of creating yet another common assembly now though.